bin/changelog

```diff
@@ -0,0 +1,303 @@
+    framework: input.framework
+  };
+};
+
+const createEntryCommand = async(args) => {
+  // https://stackoverflow.com/a/11958481
+  const gitBranchName = (await execa('git', ['rev-parse', '--symbolic-full-name', '--abbrev-ref', 'HEAD'])).stdout;
+  const maybeExtractedNumber = extractNumberFromString(gitBranchName);
+
+  const answers = isTTY ? await inquirer.prompt([
+    {
+      name: 'title',
+      type: 'input',
+      message: 'Title of the entry (e.g. "Fixed an issue with...")',
+      validate: s => s.length > 0,
+      when: typeof args.title === 'undefined'
+    },
+    {
+      name: 'issue',
+      type: 'input',
+      message: '# of the related issue/PR (e.g. 512)',
+      default: maybeExtractedNumber,
+      validate: s => ((Number.isNaN(Number(s)) || s.length === 0) ? 'The provided input is not a number' : true),
+      filter: s => Number(s),
+      when: typeof args.issue === 'undefined'
+    },
+    {
+      name: 'type',
+      type: 'list',
+      message: 'Type of the change',
+      choices: changelogEntryTypes,
+
+      default: ({ title }) =>
+        // Attempts to find a suitable default choice based on the title.
+        // e.g. the title "Fixed every bug" would return the type "fixed",
+        // and "Added 10 features" would return "added".
+        changelogEntryTypes.find(
+          type => title.toLowerCase().includes(type)
+        ) || 'changed',
+
+      when: typeof args.framework === 'undefined'
+    },
+    {
+      name: 'framework',
+      type: 'list',
+      message: 'Framework',
+      choices: changelogFrameworkTypes,
+      default: 'none',
+      when: typeof args.framework === 'undefined'
+    }
+  ]) : {};
+
+  const changelogEntry = assertChangelogEntryFormat({ ...args, ...answers });
+
+  const filename = changelogEntry.issue.toString();
+  const destination = entryDestination(filename);
+
+  console.log(`
+${chalk.dim(
+    `Your entry will be saved in ${chalk.reset.blue.underline(destination)}.
+This is how it will look like compiled to markdown in ${chalk.reset.blue.underline('CHANGELOG.md')}:`
+  )}
+
+${chalk.yellow.italic(`### ${uppercaseFirstLetter(changelogEntry.type)}
+- ${stringifyChangelogEntryObject(changelogEntry)}`)}
+`);
+
+  await askToProceedOrExit();
+
+  if (await fileExists(destination)) {
+    console.log(`The file \`${destination}\` already exists.`);
+
+    await askToProceedOrExit('Overwrite?');
+  }
+
+  await fs.writeFile(destination, JSON.stringify(changelogEntry, null, 2));
+
+  console.log();
+  console.log(`Entry created! Run ${chalk.blackBright(`\`rm ${destination}\``)} to undo.`);
+};
+
+const consumeCommand = async(args) => {
+  const { dryRun } = args;
+
+  const formattedDefault = args.date === undefined
+    ? moment(hotConfig.HOT_RELEASE_DATE, 'DD/MM/YYYY').format('YYYY-MM-DD')
+    : args.date;
+
+  const { date } = isTTY ? await inquirer.prompt([
+    {
+      name: 'date',
+      type: 'input',
+      message: 'When should the version be released?',
```

Comment:
```suggestion
      message: 'When should the version be released? (YYYY-MM-DD)',
```
I know that the consume command generally will be used within CI. But, maybe it's a good idea to add the preferred date format in parentheses in the case when the developer wants to do that?

Reply:
There is a default format being shown already, unless you mean something else?
<img width="643" alt="image" src="https://user-images.githubusercontent.com/7091471/103889517-d91a1f00-50e6-11eb-8e51-91f82d384993.png">

Reply:
True, but what if someone wants to consume the logs and the read release date is `2020-10-10` 😄 ? It's a minor issue (if at all). So we can leave it as it is.

Reply:
True, I didn't notice that you actually spelled out the whole format, my bad. Let's hope that nobody assumes `YYYY-DD-MM` then, haha.